portfolio/src/main/webapp/index.html

```diff
@@ -18,7 +18,7 @@ <h2>Background</h2>
          and algorithms course at my university. I absolutely love it 
          because I get to help people learn about my favorite subject 
          and I find it extremely rewarding helping students finally 
-         make sense for students. 
+         understand the material. 
```

Comment:
nit: to help students understand the course material.

Reply:
I reworded it. Thanks!